.eslintrc

```diff
@@ -215,6 +215,7 @@
     "@wordpress/no-unused-vars-before-return": ["error", {
       "excludePattern": "^use"
     }],
+    "@wordpress/data-no-store-string-literals": "error",
```

Comment:
Neat! Forgot that there was a lint rule for this.

Let's add this on line ~600 though so it's only enabled for the stories-block package et al